src/pages/api/autocomplete.ts

```diff
@@ -0,0 +1,52 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { z, ZodError } from "zod";
+import { MasterItem, List, Prisma, PrismaClient } from "@prisma/client";
+var stringSimilarity = require("string-similarity");
```

Comment:
this needs to be added as a dependency to the package.json file

Reply:
Added the dependency